client/src/main/scala/skuber/api/package.scala

```diff
@@ -376,19 +376,32 @@ package object client {
        // if this is a POST we don't include the resource name in the URL
        val nameComponent: Option[String] = method match {
          case HttpMethods.POST => None
-         case _ => Some(obj.name)
+         case _                => Some(obj.name)
        }
        modify(method, obj, nameComponent)
      }
 
      private[skuber] def  modify[O <: ObjectResource](method: HttpMethod, obj: O, nameComponent: Option[String])(
        implicit fmt: Format[O], rd: ResourceDefinition[O], lc: LoggingContext): Future[O] =
      {
+
+       // Object namespace has higher priority.
```

Comment:
In this case I think the code is fairly straightforward and explained in the issue, so I think something like the following comment is enough:
```
      // Namespace set in the object metadata (if set) has higher priority than that of the
      //  request context (see Issue #204)
       val targetNamespace = if (obj.metadata.namespace.isEmpty) namespaceName else obj.metadata.namespace
```

Reply:
ack